protocols/v2/roles-logic-sv2/src/job_creator.rs

```diff
@@ -254,14 +254,14 @@ impl JobsCreators {
 // Test
 #[cfg(test)]
 
-mod tests {
+pub mod tests {
     use super::*;
     use binary_sv2::u256_from_int;
     use bitcoin::{secp256k1::Secp256k1, Network};
     use quickcheck::{Arbitrary, Gen};
     use std::{borrow::BorrowMut, cmp, vec};
 
-    pub fn from_gen(g: &mut Gen) -> NewTemplate<'static> {
+    pub fn template_from_gen(g: &mut Gen, id: u64) -> NewTemplate<'static> {
```

Comment:
Why you need to pass the id here?

Reply:
no reason, I think this was from resolving conflicts on outdated code. resolving it now